integration-test/background/click-to-load-youtube.js

```diff
@@ -40,10 +43,14 @@ function summariseYouTubeRequests (requests) {
             stats = youTubeStandard
         } else if (youTubeNocookieDomains.has(request.url.hostname)) {
             stats = youTubeNocookie
+        } else if (youTubeImageDomains.has(request.url.hostname)) {
+            stats = youTubeImage
         } else {
             continue
         }
 
+        hasYouTubeAutoPlay = hasYouTubeAutoPlay || request.url.searchParams.get('autoplay') === '1'
```

Comment:
Nit: Perhaps put this inside the `if (youTubeStandardDomains.has(request.url.hostname)) {` condition (or whichever is the correct one)?

Nit: I would prefer `if (request.url.searchParams.get('autoplay') === '1') hasYouTubeAutoPlay = true`, but up to you.